src/command-registry.coffee

```diff
@@ -253,7 +254,10 @@ class CommandRegistry
       # registration order.
       for listener in listeners by -1
         break if immediatePropagationStopped
-        listener.callback.call(currentTarget, dispatchedEvent)
+        if typeof listener.listener.handleEvent is 'function'
```

Comment:
Can we tweak the names here to avoid the `listener.listener` thing? We need a noun that describes the object that contains the listener... `commandMetadata`? Just `command`?